Swissdox/Swissdox.py

```diff
@@ -38,15 +34,23 @@ def __init__(self,
             self.default_folders = ["raw-data", "intermediate-data"]
 
         self.DEFAULT_SCHEMA_VERSION = "1.2"
-        self.SCHEMA_CONFIG_PATH = os.path.normpath(os.path.join(Path(__file__).parent.resolve(), "schemas/available-schemas.yaml"))
+        self.SCHEMA_CONFIG_PATH = os.path.normpath(
+            os.path.join(Path(__file__).parent.resolve(), "schemas/available-schemas.yaml"))
 
     def initialize_dataset(self, data_root_folder, validate_query_file=True, query_string=None):
```

Comment:
I think it would me beneficial to add typehints -> data_root_folder:str, validate_query_file:bool=True, query_string:str=None etc.

Reply:
ist angepasst